README.rst

```diff
@@ -120,22 +118,28 @@ Get table from CARTO, make changes in pandas, sync updates with CARTO:
 
 .. code:: python
 
-    import cartoframes
+    from cartoframes.auth import set_default_context
+    from cartoframes.data import Dataset
+
     # `base_url`s are of the form `https://{username}.carto.com/` for most users
-    cc = cartoframes.CartoContext(base_url='https://eschbacher.carto.com/',
-                                  api_key=APIKEY)
+    set_default_context(
+        base_url='https://your_user_name.carto.com/',
+        api_key='your api key'
+    )
+
+    # create a dataset object
+    d = Dataset.from_table('brooklyn_poverty_census_tracts')
 
     # read a table from your CARTO account to a DataFrame
-    df = cc.read('brooklyn_poverty_census_tracts')
+    df = d.download()
 
-    # do fancy pandas operations (add/drop columns, change values, etc.)
+    # perform operations on you dataframe
     df['poverty_per_pop'] = df['poverty_count'] / df['total_population']
 
     # updates CARTO table with all changes from this session
-    cc.write(df, 'brooklyn_poverty_census_tracts', overwrite=True)
-
+    d.upload(df, if_exists='replace')
 
-.. image:: https://raw.githubusercontent.com/CartoDB/cartoframes/master/docs/img/read_demo.gif
+.. image:: https://raw.githubusercontent.com/CartoDB/cartoframes/master/docs/img/data-workflow.gif
 
 Write an existing pandas DataFrame to CARTO.
```

Comment:
This cc.write example is covered with the previous one (d.upload), right?

Reply:
Correct! Removing :)

Reply:
It also made me look at the upload example and realize I made some mistakes